coala_langserver/langserver.py

```diff
@@ -108,7 +108,11 @@ def start_tcp_lang_server(bind_addr, port, handler_class):
         {'DELEGATE_CLASS': handler_class}
     )
 
-    server = socketserver.TCPServer((bind_addr, port), wrapper_class)
+    try:
```

Comment:
This change should be a new PR.

Ideally create a new issue showing why this change is needed.